apis/Google.Cloud.Spanner.Data/docs/history.md

```diff
@@ -1,5 +1,25 @@
 # Version history
 
+# Version 3.9.0, released 2021-06-09
+
+- [Commit 0fb438e](https://github.com/googleapis/google-cloud-dotnet/commit/0fb438e): feat(spanner): add `query_optimizer_statistics_package` support (see below)
+
+The optimizer statistics package can be set through `QueryOptions`, which can be configured through the following mechanisms:
+
+- At the `SpannerConnection` level.
+- Through the `SPANNER_OPTIMIZER_STATISTICS_PACKAGE` environment variable.
+- At a query level.
+
+If the options are configured through multiple mechanisms then:
+
+- Options set at an environment variable level will override options configured at the `SpannerConnection` level.
```

Comment:
Oh is this backwards. It's a little unexpected if it's right. Specially given that the one at the query level overrides both environment and connection, so this is neither a top-down nor bottom-up approach.

Reply:
Hmm... indeed. Okay, will get back to Shanika to check that.

Reply:
Yup, I see why it might be confusing. But this is what the feature team insisted should be the behavior. It works the same way for the other optimizer version field that we added early last year and this one needs to be consistent with that.

Reply:
Okay, thanks for the confirmation - will merge on that basis.